python3.bork: 8.0.0 → 9.0.0

nicoonoclaste opened the pull request:
## Description of changes

Update `bork` to v9.0.0, replacing dependency on `twine` with its own `urllib3`-based uploader.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] Tested compilation of all affected packages using `nixpkgs-review`.
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

nicoonoclaste committed:
python3.bork: 8.0.0 → 9.0.0

gador commented:
Thanks for the update. Could you also add the license (seems to be MIT)  to the `meta` field?

nicoonoclaste committed:
bork: add missing `meta.license` attribute

nicoonoclaste committed:
python3.bork: use python's default `meta.platforms`

`platforms.all` is not technically correct, as CPython itself
is not useable on all the platforms defined in nixpkgs.

gador commented:
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 331546` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>python311Packages.bork</li>
<li>python311Packages.bork.dist</li>
<li>python312Packages.bork</li>
<li>python312Packages.bork.dist</li>
</ul>
</details>
